oak-run/src/test/java/org/apache/jackrabbit/oak/plugins/tika/CSVFileBinaryResourceProviderTest.java

```diff
@@ -50,12 +52,24 @@ public void testGetBinaries() throws Exception {
 
         CSVFileBinaryResourceProvider provider = new CSVFileBinaryResourceProvider(dataFile, new MemoryBlobStore());
 
-        Map<String, BinaryResource> binaries = provider.getBinaries("/").uniqueIndex(BinarySourceMapper.BY_BLOBID::apply);
+        Map<String, BinaryResource> binaries = StreamUtils.toStream(provider.getBinaries("/")).collect(Collectors.toMap(
```

Comment:
There is no `uniqueIndex` API on Apache's `FluentIterable`, so I had to transform to `Stream` to create a `Map`.